src/System.Resources.ReaderWriter/src/System/Resources/ResourceReader.cs

```diff
@@ -389,99 +462,5 @@ private int Read7BitEncodedInt()
             } while ((b & 0x80) != 0);
             return count;
         }
-
-        internal sealed class ResourceEnumerator : IDictionaryEnumerator
```

Comment:
Oh, you're not even implementing IDictionaryEnumerator on the struct Enumerator.  Nevermind.